### DISCUSSION TITLE
Adding Playlist and Podcast option

### DISCUSSION THREAD

**Discussion by UNKN0WN006** [tanish35/Pomodoro-App#39] [Announcements]
**_### Issue suggestion:_**
For example, I am attending your application, and while in the study session, I want to listen to some music from any source. So, what would I do?  I will look to see if there's any option to play a playlist or if Spotify is connected to the application so that while studying too, I don't have to change the tab to search for a playlist and play it.
### 
**_Solution:_**
I would suggest adding a repo where the playlist location or the podcast location is on the upper right-hand corner of the timer screen where saved playlists are located. And the user could play that while being in the study sessions. What can you do is embed playlists and podcasts into the website using the src command to embed a playlist. You can do it by using this code:

<iframe src="https://embed.spotify.com/?uri=spotify:user:_michaelgomez_:playlist:2lytqp5Qmr75B2qymGozdk" width="300" height="380" frameborder="0" allowtransparency="true"></iframe>

**_Implementation:_**
It would help in keeping a good image of the session to study as well as listen to study Lofi beats or podcasts to strengthen my motivation towards studying

Other Suggestions:
Other things look great though I think the display box for the study session could have been larger.

**Comment by tanish35** [comment]
Feature has been added

Also what does adding a repo mean ?

Future:
Link to be given by user

**Comment by tanish35** [comment]
Future feature also implemented
